core/settings/qPrefPrivate.h

```diff
@@ -22,15 +22,17 @@ class qPrefPrivate {
 	qPrefPrivate() {}
 };
 
+// helper function to ensure there's a '/' between group and name
+QString keyFromGroupAndName(QString group, QString name);
```

Comment:
I add "extern" in front, instead of just being a forward declaration....but that might be a matter of taste.